packages/dbml-core/src/parse/ANTLR/parsers/postgresql/PostgreSQLLexerBase.js

```diff
@@ -0,0 +1,23 @@
+import antlr4 from 'antlr4';
+
+// original (C#): https://github.com/antlr/grammars-v4/blob/master/sql/postgresql/CSharp/PostgreSQLLexerBase.cs
+// Base class required by the generated lexer
+export default class PostgreSQLLexerBase extends antlr4.Lexer {
+  constructor (input) {
+    super(input);
+  }
```

Comment:
eslint error: `Useless constructor`

Reply:
Fixed in https://github.com/holistics/dbml/pull/416/commits/5aad86bc89dac9fa4fa9180630cea993815d1b16